### PR TITLE
Reduce uggs

### LINKED ISSUE #14: Hide 'name' label
This is uggs af, lets only show this to screen readers

### PR DESCRIPTION
**Pull request by zoetrope69**
Remove label visually
Only have underline for input not a box

Would close #14

Screenshot with orange border at bottom

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -77,14 +80,20 @@ input:focus {
   transition: transform .15s, opacity .15s;
 }
 
+input:focus {
+  outline: none;
+  border-bottom: 4px solid orangered;
+}
+
 label {
```
Comment (NickColley) [inline styles.css:88]: would rathre use aria-label than do this

**Review by NickColley — CHANGES_REQUESTED** [review]
hiding the label is flakier than using aria-label

**Comment by zoetrope69** [comment]
Don't know what this means, can you explain more?

**Comment by NickColley** [comment]
aria-label has pretty good support and doesnt rely on CSS hacks

**Comment by zoetrope69** [comment]
yeah i still dont know what that means though?

**Pull request closed without merging**